src/libs/analysis.js

```diff
@@ -27,7 +27,8 @@ export const launch = async ({
   try {
     await Ajax().Workspaces.workspace(namespace, name).checkBucketAccess(googleProject, bucketName, accessLevel)
   } catch (error) {
-    throw new Error('Error confirming workspace bucket access. This may be a transient problem. Please try again in a few minutes. If the problem persists, please contact support.')
+    console.error(error)
```

Comment:
As far as I know, there is no way to see the console output emitted by production code in the integration test logs (I tried `console.info`, `console.error`, etc.). From the test code itself, `console.info` is printed into the logs. Please let me know if I am wrong!

Reply:
I couldn't tell why it's being swallowed  :(

Reply:
What if, just before we click the button in the test, we register a listener? https://github.com/puppeteer/puppeteer/blob/main/docs/api.md#event-requestfinished (see further https://github.com/puppeteer/puppeteer/blob/main/docs/api.md#class-httprequest)

Reply:
Thanks @zarsky-broad! I found it difficult to get the details of the object when using the console listener, but the request listener is giving the same level of detail that I would have gotten with my previous commit. So now this debugging only impacts the test.

Reply:
Example logging: https://app.circleci.com/pipelines/github/DataBiosphere/terra-ui/7553/workflows/20c242b2-0ecb-41b5-805f-759ca7bc48de/jobs/35473/parallel-runs/0/steps/0-109

Reply:
Where exactly are we looking on the link above for the example of the logging?

Reply:
1. Click "Details" next to the integration job in checks passed section.
2. Find the parallel run that included the run-workspace job (Usually the longest running one).
3. Expand the stage above "Uploading test results".
![image](https://user-images.githubusercontent.com/484484/146612674-fe681f84-338a-4b08-995c-23da1c281943.png)